homework-submission/react-hyf/server/server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const path = require('path');
+const buildFolder = '../client/build';
+const app = express();
+
+app.use(express.static(path.join(__dirname, buildFolder)));
+
+app.get('/', (req, res) => {
+  res.sendFile(express.static(path.join(__dirname, buildFolder, 'index.html')));
+});
+
+const PORT = 4000;
```

Comment:
Use `const PORT = process.env.PORT || 4000` otherwise you won't be able to host it.

Host providers return a dynamic port